client/state/happychat/selectors.js

```diff
@@ -89,3 +92,12 @@ export const canUserSendMessages = createSelector(
 	),
 	[ getHappychatConnectionStatus, getHappychatChatStatus ]
 );
+
+export const getHappychatLastActivityTimestamp = state => state.happychat.lastActivityTimestamp;
+
+export const wasHappychatRecentlyActive = state => {
```

Comment:
@jordwest In your last work on this, [you returned `false` if the last active time was in the future](https://github.com/Automattic/wp-calypso/pull/11604/files#diff-af13124b52ffdb8d7d913a442be42611R34). What's the reasoning for that? I couldn't figure it out so I omitted it but wanted to check with you.